### DISCUSSION TITLE
Objects not show up

### DISCUSSION THREAD

**Discussion by tsing-xu** [CollectionBuilder/collectionbuilder.github.io#126] [General]
I followed the tutorial, but the objects are not showing.

**Comment by evanwill** [comment]
@tsing-xu in your project's [_config.yml](https://github.com/tsing-xu/ui-chinese-composers/blob/main/_config.yml) you have the value 
`metadata: datacollection`
which does not match any CSV name in your "_data" folder. 
You have one CSV named ["data_collection.csv"](https://github.com/tsing-xu/ui-chinese-composers/blob/main/_data/data_colletion.csv), to use that you would have to edit your _config.yml to look like:
`metadata: data_collection`

Note that in the "data_collection.csv" it looks like you have PDF items in the "filename" column. You will want to update the "format" column values to match the file type. Currently it has `image/jpeg` which you will have to update to `application/pdf` for PDF items.
